### PR TITLE
Fix an exception with 0-size test cases in the viewer handler.

### PR DESCRIPTION
**Pull request by mbarbella-chromium**
PTAL. We could do a more generic fix in storage.read_data but it would require us to check the size in all cases, which seems like it could hurt performance in cases where we don't expect issues. Since we already know the size in the handler this seems like the best quick fix to me, but I'm open to either option.

### CODE REVIEW

File: src/appengine/handlers/viewer.py [hunk 1]
```diff
@@ -45,13 +45,20 @@ def get(self):
...
-    try:
-      content = unicode(blobs.read_key(key), errors='replace')
-    except Exception:
-      raise helpers.EarlyExitException('Failed to read content.', 400)
+    # TODO(mbarbella): Workaround for an issue in the Cloud Storage API. Remove
+    # once it is fixed properly upstream:
+    # https://github.com/googleapis/google-cloud-python/issues/6572
+    if blob_size:
+      try:
+        content = unicode(blobs.read_key(key), errors='replace')
```
Comment (inferno-chromium) [inline src/appengine/handlers/viewer.py]: maybe remove this unicode thing along the way and fix https://bugs.chromium.org/p/chromium/issues/detail?id=920160 :) stage to make sure it works.

Comment (mbarbella-chromium) [reply src/appengine/handlers/viewer.py]: SGTM.

Comment (mbarbella-chromium) [reply src/appengine/handlers/viewer.py]: Seems to cause issues later in template rendering, which I guess explains why we had the hack initially. I'm going to leave it out of this change.

**Pull request merged**